mgmt/rest/server.go

```diff
@@ -92,6 +93,15 @@ func New(cfg *Config) (*Server, error) {
 		negroni.HandlerFunc(s.authMiddleware),
 	)
 	s.r = httprouter.New()
+
+	// Deals with CORS
+	c := cors.New(cors.Options{
+		AllowedOrigins: []string{"*"},
```

Comment:
I don't think that it's safe to allow unrestricted access from any domain. Is there a possibility to constrain allowed origins - via snapteld configuration maybe?